### PR TITLE
Renaming & Blocks

### PR DESCRIPTION
**Pull request by HellZDarkArkon**
renamed "Castle Brick" to "Chess Brick"

created "Coarse Stone Slab", "Flowery Mossy Stone Brick Slab", "Mycelium Brick Slab", "Frost Brick Slab", "Shade Brick Slab" and "Flowery Mossy Stone Brick Stairs"

### CODE REVIEW

**Comment by kirderf1** [comment]
Alright, so this PR now has more stuff than the castle -> chess renaming. Please update the PR title and description accordingly.

**Comment by HellZDarkArkon** [comment]
Changed Registry name back to castle_bricks

**Review by kirderf1 — COMMENTED** [review]
Just one minor thing that I don't think should be there.

I haven't looked into the asset files much as I doubt there would be any easy-to-spot issues with them. I may want to try running the branch once before merging to make sure that minecrafts model loader doesn't have anything to complain about, though I suspect that you've already made sure on your end that there are no issues.

File: src/main/java/com/mraof/minestuck/block/MSBlocks.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.registries.IForgeRegistry;
 import net.minecraftforge.registries.ObjectHolder;
+import software.bernie.shadowed.eliotlash.mclib.math.functions.classic.Abs;
```
[the remark this answers is not in the document]
Comment (HellZDarkArkon) [reply src/main/java/com/mraof/minestuck/block/MSBlocks.java:28]: yeah, not sure why. it appeared shortly after i cloned the repo.

**Pull request merged**